vignettes/how_to_harmonize.Rmd

```diff
@@ -0,0 +1,120 @@
+
+```{r ,eval=F, results = "hide"}
+# Harmonize individual datasets
+age_sex2001 <- rec_with_table(cchs2001_p, c("DHHGAGE_cont", "DHH_SEX"))
+age_sex2003 <- rec_with_table(cchs2003_p, c("DHHGAGE_cont", "DHH_SEX"))
+age_sex2005 <- rec_with_table(cchs2005_p, c("DHHGAGE_cont", "DHH_SEX"))
+age_sex2007_2008 <- rec_with_table(cchs2007_2008_p, c("DHHGAGE_cont", "DHH_SEX"))
+age_sex2009_2010 <- rec_with_table(cchs2009_2010_p, c("DHHGAGE_cont", "DHH_SEX"))
+age_sex2011_2012 <- rec_with_table(cchs2011_2012_p, c("DHHGAGE_cont", "DHH_SEX"))
+age_sex2013_2014 <- rec_with_table(cchs2013_2014_p, c("DHHGAGE_cont", "DHH_SEX"))
+age_sex2015_2016 <- rec_with_table(cchs2015_2016_p, c("DHHGAGE_cont", "DHH_SEX"))
+age_sex2017_2018 <- rec_with_table(cchs2017_2018_p, c("DHHGAGE_cont", "DHH_SEX"))
+
+# Merge harmonized data
+combined_age_sex <- merge_rec_data(age_sex2001, age_sex2003, age_sex2005, age_sex2007_2008, age_sex2009_2010, age_sex2011_2012, age_sex2013_2014, age_sex2015_2016, age_sex2017_2018)
+
+```
+
+## How to combine all variables in the variable_details sheet across multiple cycles
+
+To combine a large number of variables, it is best to use `variables.csv` and `variable_details.csv`. There are vignettes that further describe variables and variable_details, including how to add or customize transformed variables.
+
+### Option 1: Using _cchsflow_ variable_details sheet
+
+When the variable argument in `rec_with_table()` is not specified, all variables listed in `variables.csv` and `variable_details.csv` will be transformed. In this example, all variables from the _cchsflow_ `variables.csv` and `variable_details.csv` sheets from 2001 to 2018 CCHS datasets will be transformed and labeled using  `rec_with_table()`, which is then combined into one dataset and labeled using  `merge_rec_data()`.
+
+```{r ,eval=F, results = "hide"}
+# Harmonize individual datasets
+harmonized_2001 <- rec_with_table(cchs2001_p)
+harmonized_2003 <- rec_with_table(cchs2003_p)
+harmonized_2005 <- rec_with_table(cchs2005_p)
+harmonized_2007_2008 <- rec_with_table(cchs2007_2008_p)
+harmonized_2009_2010 <- rec_with_table(cchs2009_2010_p)
+harmonized_2011_2012 <- rec_with_table(cchs2011_2012_p)
+harmonized_2013_2014 <- rec_with_table(cchs2013_2014_p)
+harmonized_2015_2016 <- rec_with_table(cchs2015_2016_p)
+harmonized_2017_2018 <- rec_with_table(cchs2017_2018_p)
+
+# Merge harmonized data
+combined_all_cycles <- merge_rec_data(harmonized_2001, harmonized_2003, harmonized_2005, harmonized_2007_2008, harmonized_2009_2010, harmonized_2011_2012, harmonized_2013_2014, harmonized_2015_2016, harmonized_2017_2018)
+```
+
+### Option 2: Using your own variable_details sheet
+
+In this example, all variables from personalized `variables.csv` and `variable_details.csv` sheets from 2001 to 2018 CCHS datasets will be transformed and labeled using  `rec_with_table()`, which is then combined into one dataset and labeled using  `merge_rec_data()`.

```

Comment:
I would consider showing the relationship between `variables.csv` and `sample_variables` and `variable_details.csv` and `sample_variable_details`. Is the user expected to do something like `sample_variables <- readr::read_csv('variables.csv')` in their workspace before using the personalized spreadsheets?